src/test/scala/com/databricks/spark/xml/XmlSuite.scala

```diff
@@ -207,26 +207,23 @@ class XmlSuite extends FunSuite with BeforeAndAfterAll {
   test("DDL test") {
     sqlContext.sql(
       s"""
-         |CREATE TEMPORARY TABLE carsTable
+         |CREATE TEMPORARY TABLE carsTable1
```

Comment:
This started failing in Spark 2.3 as the temp table already existed across examples.